emcc

```diff
@@ -1407,6 +1422,9 @@ try:
       final += '.mem.js'
       src = None
 
+  if pthreads:
+    shutil.copyfile(shared.path_from_root('src', 'pthread-main.js'), os.path.join(os.path.dirname(os.path.abspath(target)), 'pthread-main.js'))
```

Comment:
we should document this adding of the file in the `site/` dir somewhere. it's another file people will need to copy around etc.